plugins/outputs/stomp/stomp_test.go

```diff
@@ -0,0 +1,60 @@
+package stomp
+
+import (
+	"math/rand"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/go-stomp/stomp"
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf/plugins/serializers"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func TestConnectAndWrite(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration test in short mode")
+	}
+
+	port := rand.Intn(7000-3000) + 3000
```

Comment:
you are grabbing some random port?

Is it possible to use testcontainers, like we do with other integration tests to launch an ActiveMQ instance and write to it?

as-is are these tests even useful?

Reply:
I updated the test to use test containers